pset-iterators.js

```diff
@@ -211,6 +270,24 @@ console.log('Problem 8:')
 
 // Add your code below this line
 
+const firstNames = presidents.map((singularPresident) => {
+  return singularPresident.president.split(" ")[0]
+})
+
+console.log(firstNames)
+
+let firstNameCount = {}
+  firstNameCount = firstNames.reduce((nameCount, name) => {
```

Comment:
Very good use of the reduce method to fill an empty object! You could also pass the initial value of first parameter after you define the conditions of the method. [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L259) is an example of that.